quickstart/configs.go

```diff
@@ -22,15 +24,22 @@ func configBaseForm(quickstart *Quickstart) (*State, error) {
 		configFields := []huh.Field{
 			huh.NewInput().
 				Title("Choose an sdkClassName for your target:").
+				Placeholder("idiomatic name for your SDK object").
```

Comment:
given that this prompt is given for every language, I think we need to better explain what this actually does. Maybe it should say `Name your SDK object: _if you choose "sdk", your users will write "sdk.doThing()"_` or something to that effect